glossary additions (#6134)
GoogleChrome/developer.chrome.com: site/en/_partials/privacy-sandbox/glossary-entries/scaling.md

Author: added "in the context of the Attribution Reporting API,"

Author: @samdutton can you check this? I can delete if you think I should. thanks!

> [Scaling](/docs/privacy-sandbox/private-aggregation-fundamentals/#noise-and-scaling) refers to multiplying your aggregatable value by a number you choose to adjust the impact of noise. It impacts the effect of noise and the use of your contribution budget.

Reviewer: We may want to drop "Scaling". For the most part, we use it in reference to aggregation, but it could also be used in other enlarging of infrastructure.

Author: yes was not sure about this

Author: resolving this comment and re-asking so Sam can find easily

> [Scaling factor](/docs/privacy-sandbox/private-aggregation-fundamentals/#noise-and-scaling), in the context of the Attribution Reporting API, refers a value you choose to  multiply your aggregatable value by to adjust the impact of noise. This process impacts the effect of noise and the use of your contribution budget. The [noise lab](https://noise-lab.uc.r.appspot.com/?mode=simple) lets you try out combinations.

Reviewer: ```suggestion
[Scaling factor](/docs/privacy-sandbox/private-aggregation-fundamentals/#noise-and-scaling), in the context of the Attribution Reporting API, refers an amount by which you choose to multiply an aggregatable value. Scaling impacts the effect of noise and your contribution budget.
```

Removing Noise Lab as it should exist within the confines of ARA.
